<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_BaseAngle</class>
 <widget class="QWidget" name="Dlg_BaseAngle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Line1</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ua</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ub</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Uc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ib</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>32</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Line2</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ua</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ub</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Uc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ib</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>。</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>▲α</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>。</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
   </property>
   <property name="text">
    <string>获取角度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
